examples/nni_data_augmentation/README.md

```diff
@@ -0,0 +1,35 @@
+## Tuun data augmentation
+##### Installation
+###### This example
```

Comment:
- Could we be explicit here on which directory should be in at start of installation?
- Do we have a setup.py in this directory (seems to be called below)?

Reply:
Added "First cd into this folder" to the beginning of the instructions and deleted setup.py usage